.github/workflows/build.yml

```diff
@@ -204,15 +249,15 @@ jobs:
 
   # Build and push manifest
   manifest:
-    uses: databox/.github/.github/workflows/manifest.yml@master
+    uses: databox/.github/.github/workflows/manifest.yml@cdn-support
```

Comment:
to je verjetno samo za testing?

Reply:
Y, to je za namene testinga. Sem [odstranil](https://github.com/databox/.github/pull/9/commits/eb876bb061c3b8b5c6df75b0a711465c4e8bf6cb) pred mergem